packages/fether-react/src/stores/createAccountStore.js

```diff
@@ -208,10 +214,17 @@ export class CreateAccountStore {
   };
 
   @action
-  importFromSigner = async ({ address, chainId }) => {
-    await this.clear();
+  importFromSigner = async ({ address, signerChainId }) => {
+    /**
+     * If we do not comment out `await this.clear()` here, then in the next step is
+     * `componentDidMount` of `AccountName` component, the value of `createAccountStore.address`
+     * and `createAccountStore.signerChainId` will be both undefined, even though we tried
```

Comment:
that's weird

Reply:
maybe try again if this.clear() doesn't reset isimport ?

Reply:
it was because I was supplying it as `await importFromSigner({ address, signerChainIdInt });` and then receiving it as `importFromSigner = async ({ address, signerChainId }) => {`.
So I've just renamed it to `signerChainId` to avoid confusion